frontend/src/app/testResults/uploads/CsvSchemaDocumentation.scss

```diff
@@ -40,3 +40,19 @@
     font-style: normal;
   }
 }
+
+.csv-section-tablist {
+  li:first-child {
+    padding-left: 0;
+  }
+
+  .tab-button {
+    display: inline-block;
+  }
+
+  .tab-selected,
+  .tab-button:hover {
+    border-bottom: 0.25rem solid #005ea2;
+    padding-bottom: 0.5rem;
```

Comment:
We discussed this during our pairing session about adding a `cursor: pointer;` to make a visual distinction that this tab is clickable.